NEWS

```diff
@@ -60,6 +62,7 @@ Sourav Singh (first contribution)
 Steve Bond
 Uri Laserson
 Vincent Davis
+Adam Kurkiewicz (first contribution)
```

Comment:
I forget why now, but we list the contributors in the NEWS file alphabetically by first name - so you'd currently be first :)